app/ExposureHistoryContext.tsx

```diff
@@ -88,6 +88,10 @@ const ExposureHistoryProvider: FunctionComponent<ExposureHistoryProps> = ({
     return subscription.remove;
   }, [exposureInfoSubscription, toExposureHistory]);
 
+  useEffect(() => {
+    getCurrentExposures();
+  }, [toExposureHistory]);
```

Comment:
I believe that this one will give you a warning that it needs the signature of the callback also as a dependency, sounds weird but I remember seeing such warnings 😅  probably when you do that the linter will tell you that the function needs to be defined before you use it so moving the method declaration before this hook is likely something you'll need to do 😬